motion/thruster_interface_auv/include/thruster_interface_auv/thruster_interface_auv_driver.hpp

```diff
@@ -15,15 +15,22 @@
 #include <string>
 #include <vector>
 
-#define IDLE_PWM_VALUE 1500
+#define IDLE_PWM_VALUE 1500  ///<-- pwm value to output when force = 0.00
```

Comment:
Another also; is it necessary to define this as a constant and not pass it as a value from the config file?